mini-cloudinary.rb

```diff
@@ -87,11 +88,12 @@ class App < Sinatra::Base
         width = params['width']
         height = params['height']
 
-        ret_value = mc.handle_request(url, width, height)
-        if ret_value.is_a?(String)
-            send_file(file_path) 
-        else
-            ret_value
+        begin
+            send_file(mc.handle_request(url, width, height))
```

Comment:
I feel that `MiniCloudinary` shouldn't be familiar with the term `request` at all. `request` is the domain of the web application. 

Think about it as a black box which knows how to transform a given input based on the given parameters.
For example, calling `mc.transform` or `mc.lpad` makes more sense to me.